AutomatedTesting/Gem/PythonTests/editor/TestSuite_Main_Optimized.py

```diff
@@ -73,3 +73,7 @@ class test_Menus_ViewMenuOptions_Work(EditorSharedTest):
     @pytest.mark.skip(reason="Times out due to dialogs failing to dismiss: LYN-4208")
     class test_Menus_FileMenuOptions_Work(EditorSharedTest):
         from .EditorScripts import Menus_FileMenuOptions as test_module
+
+    
+    class test_Sample_Editor_Tests(EditorSharedTest):
```

Comment:
The name of the test class still needs an update to test_BasicEditorWorkflows_ExistingLevel_EntityComponentCRUD